src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -231,6 +247,73 @@ function ReportActionItemSingle({
             </UserDetailsTooltip>
         );
     };
+
+    const getHeading = useCallback(() => {
+        return () => {
+            if (displayAllActors && personArray.length === 2 && isReportPreviewAction) {
+                return (
+                    <View style={[styles.flexRow]}>
+                        <ReportActionItemFragment
+                            style={[styles.flex1]}
+                            key={`person-${action?.reportActionID}-${0}`}
+                            accountID={actorAccountID ?? -1}
+                            fragment={{...personArray.at(0), type: 'TEXT', text: displayName ?? ''}}
+                            delegateAccountID={action?.delegateAccountID}
+                            isSingleLine
+                            actorIcon={icon}
+                            moderationDecision={getReportActionMessage(action)?.moderationDecision?.decision}
+                        />
+                        <Text
+                            numberOfLines={1}
+                            style={[styles.chatItemMessageHeaderSender, styles.pre]}
+                        >
+                            {` & `}
+                        </Text>
+                        <ReportActionItemFragment
+                            style={[styles.flex1]}
+                            key={`person-${action?.reportActionID}-${1}`}
+                            accountID={parseInt(`${secondaryAvatar?.id ?? -1}`, 10)}
+                            fragment={{...personArray.at(1), type: 'TEXT', text: secondaryAvatar.name ?? ''}}
+                            delegateAccountID={action?.delegateAccountID}
+                            isSingleLine
+                            actorIcon={secondaryAvatar}
+                            moderationDecision={getReportActionMessage(action)?.moderationDecision?.decision}
+                        />
+                    </View>
+                );
+            }
+            return (
+                <View>
+                    {personArray.map((fragment) => (
+                        <ReportActionItemFragment
+                            style={[styles.flex1]}
+                            key={`person-${action?.reportActionID}-${actorAccountID}`}
```

Comment:
I think it's better to use index suffix like before. Because it's possible that 
`personArray.length === 2` and `isReportPreviewAction` is false right?

Reply:
I did it because of this eslint error-

<img width="447" alt="Screenshot 2024-10-09 at 1 45 01 PM" src="https://github.com/user-attachments/assets/75b5dba2-4053-43a0-92b3-1e29b9fb13ad">

I think we might be able to use this pattern for all multiple avatar cases, but we'd need to confirm with design and it's an easy fix so I'll do that in a separate PR if so

Reply:
To clarify the above comment, we may be able to remove the `isReportPreviewAction` check entirely

Reply:
Got it. For the lint error, maybe we can just disable the lint by adding
```
// eslint-disable-next-line react/no-array-index-key
```

Reply:
Yeah I see what you mean, updating now.